apps/agentstack-cli/src/agentstack_cli/commands/agent.py

```diff
@@ -778,9 +783,9 @@ async def run_agent(
 
         initial_form_render = next(
```

Comment:
We should probably refactor this and use the extension client and parse as service extension cc @jezekra1